delta-api/src/main/java/io/cdap/delta/api/DMLEvent.java

```diff
@@ -166,9 +185,19 @@ public Builder setIngestTimestamp(long ingestTimestampMillis) {
       return this;
     }
 
+    public Builder setSourceTimestamp(long sourceTimestampMillis) {
```

Comment:
shall we mention in comments that for unordered events , source timestamp must be set ?

Or even put an ordering attribute in the event so that we can validate that this property being set for unordered events in the build() method ?

Reply:
Added comment. We don't need to put ordering attribute on each event as its property of the source. We can possibly look into the validation as a part of separate bug.

Reply:
sg